Core/src/Geometry/CuboidVolumeBuilder.cpp

```diff
@@ -114,6 +123,22 @@ std::pair<double, double> Acts::CuboidVolumeBuilder::binningRange(
       minMax.second = surfacePosMax;
     }
   }
+
+  //Use the volume boundaries as limits for the binning
+  //and check that the layers are inside the defined tracking volume
+    
+  if (minMax.first >= min_volume_boundaries(binX))
+    minMax.first = min_volume_boundaries(binX);
+  //else
+  //  ACTS_ERROR("CuboidVolumeBuilder::binningRange::The min binning range value " << minMax.first
+  //            << "is lower than the volume boundary " << min_volume_boundaries(binX));
+  
```

Comment:
These comments would probably have to be removed.

Reply:
Done